generateMetadata.js

```diff
@@ -0,0 +1,70 @@
+const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
+const fs = require('fs').promises;
+
+const template = `<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta property="og:url" content="https://juicebox.money/p/%handle%" />
+    <meta property="og:title" content="%name%" />
+    <meta property="og:description" content="%description%" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="%logoUri%" />
+    <meta property="og:image:width" content="1200" />
+    <meta property="og:image:height" content="1200" />
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="@%twitter%" />
+    <meta name="twitter:creator" content="@%twitter%" />
+    <meta name="twitter:title" content="%name%" />
+    <meta name="twitter:description" content="%description%" />
+    <meta name="twitter:image" content="%logoUri%" />
+    <title>%name%</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <script>location.replace('/#' + location.pathname)</script>
+  </body>
+</html>`;
+
+(async () => {
+  let loaded = false;
+  let projects;
+  while (!loaded) {
+    try {
+      projects = (await (await fetch('https://gateway.thegraph.com/api/6a7675cd9c288a7b9571d5c9e78d5aff/deployments/id/QmNxBy8UnUsQr3aeBgFvdFyWbTSMiTGpJbkAJzSm5m6vYf', {
```

Comment:
We could add `dotenv` as a dependency here and use `REACT_APP_SUBGRAPH_URL`:

1. `yarn add -D dotenv`.
2. Add `require('dotenv').config()` to the top of this file.

```suggestion
      projects = (await (await projects = (await (await fetch(process.env.REACT_APP_SUBGRAPH_URL, {
```

Reply:
I added these changes, but there is "fleek/build — Build timeout", but I can't see Details, I have no access.
It looks like this REACT_APP_SUBGRAPH_URL variable is not available to build for rinkeby and the script is endlessly trying to get a list of projects.

Reply:
Looks like this timeout was just due to some concurrent builds hogging our bots. that .env variable is available for rinkeby!